scripts/msg.h.template

```diff
@@ -210,6 +235,9 @@ namespace ros
 namespace serialization
 {
 
+@[if have_plugin]@
+#ifndef @(spec.package.upper())_MESSAGE_@(spec.short_name.upper())_CUSTOM_SERIALIZER
```

Comment:
I think it would be beneficial if all macro names share a common prefix which includes the term "plugin", e.g. `@(spec.package.upper())_MESSAGE_@(spec.short_name.upper())_PLUGIN_`. Maybe the "custom" part can be skipped since by providing the macro it is kind of implied that a custom version is provided?

That would name this macro: `@(spec.package.upper())_MESSAGE_@(spec.short_name.upper())_PLUGIN_SERIALIZER`.